shared/local-debug-live.js

```diff
@@ -1,4 +1,15 @@
-export const dumbFilter = ''
+// @flow
+let envJson = {}
+if (process.env.KEYBASE_LOCAL_DEBUG_JSON) {
+  try {
+    envJson = JSON.parse(process.env.KEYBASE_LOCAL_DEBUG_JSON)
+  } catch (e) {
+    envJson = {}
+    console.warn('Invalid KEYBASE_LOCAL_DEBUG_JSON:', e)
+  }
+}
+
+export const dumbFilter = envJson.dumbFilter || ''
```

Comment:
Please put the string value before the env value, so that it takes precedence when set via the file.